aries_cloudagent/core/tests/test_plugin_registry.py

```diff
@@ -442,19 +442,31 @@ async def test_register_plugin_no_versions(self):
             ClassLoader, "load_module", async_mock.MagicMock()
         ) as load_module:
             load_module.side_effect = [
-                async_mock.MagicMock(),  # module
+                {},  # module
                 None,  # routes
                 None,  # message types
                 "str-has-no-versions-attr",  # definition without versions attr
             ]
             assert self.registry.register_plugin("dummy") is None
 
+    async def test_register_plugin_has_setup(self):
+        with async_mock.patch.object(
+            ClassLoader, "load_module", async_mock.MagicMock()
+        ) as load_module:
+            load_module.side_effect = [
+                {"setup": "pressent"},  # module
```

Comment:
```suggestion
                {"setup": "present"},  # module
```

Reply:
good catch! I also realized the duck typing check would not be triggered by a dictionary, so I have changed that code to be a class object instead.